src/AppBundle/Entity/Contributor.php

```diff
@@ -151,15 +151,15 @@ private function markUpdated(): Contributor
     /**
      * Get id.
      */
-    public function getId(): int
+    public function getId(): ?int
     {
         return $this->id;
     }
 
     /**
      * Set name.
      */
-    public function setName(string $name): Contributor
+    public function setName(?string $name): Contributor
```

Comment:
Property `$name` seems to be required.
Furthermore ORM definition does not specify this field as nullable.

Reply:
Same explanation (unfortunately)